CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ No changes to highlight.
 No changes to highlight.
 
 ## Testing and Infrastructure Changes:
-No changes to highlight.
+* Remove h11 pinning by [@ecederstrand]([https://github.com/abidlabs](https://github.com/ecederstrand)) in [PR 2820]([https://github.com/gradio-app/gradio/pull/2808](https://github.com/gradio-app/gradio/pull/2820)) 
```

Comment:
Looks good to me @ecederstrand !

Reply:
I'm going to build this and install on colab as a paranoid check but should be good to merge soon!